### PR TITLE
chore: normalize configuration values (release/0.42)

### PR DESCRIPTION
**Pull request by nathanklick**
### Related Issues

- #8666

### CODE REVIEW

File: hedera-node/configuration/mainnet/application.properties [hunk 1]
```diff
@@ -1 +1,12 @@
...
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+staking.rewardRate=48630136986000
+tokens.nfts.maxAllowedMints=10_000_000
```
Comment (nathanklick) [inline hedera-node/configuration/mainnet/application.properties:12]: These are the values taken from mainnet as of Friday, September 15th which is running release `v0.40.x` code. Reviews should verify that these values are still applicable/correct for release `v0.42.x` series code.

**Review by jsync-swirlds — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by jsync-swirlds — APPROVED** [review]
As this no longer matches the _current_ contents of file 121, we should definitely verify the following:
1. these changes are correct for mainnet release 0.42
2. file 121 will be updated to match, in conjunction with the release

**Comment by jsync-swirlds** [comment]
Just fixing an error in rebase (back to 4 files changed instead of 27).

**Review by poulok — APPROVED** [review]

**Review by qnswirlds — APPROVED** [review]

File: hedera-node/configuration/mainnet/application.properties (hunk 1 above)
Comment (mhess-swl) [reply hedera-node/configuration/mainnet/application.properties]: To my knowledge, these are correct

**Comment by mhess-swl** [comment]
Force pushed to signoff some incorrectly-signed commits

**Pull request merged**